pages/data-providers/[data-provider-id]/index.js

```diff
@@ -1,3 +1,8 @@
-const Index = () => null
+import { useRedirect } from '../../_app/hooks'
 
-export default Index
+const DataProvider = () => {
+  useRedirect('overview')
```

Comment:
Maybe it's worth of redirecting on the server-side too? Just thinking out loud.

Reply:
What do you mean? Add 302 response?

I would but I do not know how to do it.

Reply:
Something like this but I haven't tested it.

https://linguinecode.com/post/how-to-redirect-on-the-server-side-with-next-js

Reply:
I created #300 for this.